go.mod

```diff
@@ -1,28 +1,26 @@
 module k8s.io/cloud-provider-aws
 
-go 1.20
+go 1.21
```

Comment:
Are we running 1.21 yet, is this going to be an issue?

Reply:
We are using 1.21 https://github.com/openshift/cloud-provider-aws/blob/8c604228325114d969bc78ac7d98f5def4389d3e/.ci-operator.yaml#L4
So that's fine.